src/general/Mutations.cs

```diff
@@ -350,19 +352,23 @@ private OrganelleDefinition GetRandomOrganelle(bool isBacteria)
 
                 for (int side = 1; side <= 6; ++side)
                 {
-                    // Offset by hex offset
-                    result.Position = pos + Hex.HexNeighbourOffset[(Hex.HexSide)side];
-
-                    // TODO: checking one or two extra hexes in the direction would make this succeed more often
-
-                    // Check every possible rotation value.
-                    for (int rotation = 0; rotation <= 5; ++rotation)
+                    for (int multiplier = 1; multiplier <= 3; ++multiplier)
                     {
-                        result.Orientation = rotation;
-
-                        if (existingOrganelles.CanPlace(result))
+                        // Offset by hex offset multiplied by a factor to check for greater range
+                        var hexOffset = Hex.HexNeighbourOffset[(Hex.HexSide)side];
+                        hexOffset.Q *= multiplier;
+                        hexOffset.R *= multiplier;
```

Comment:
Is there no `hexOffset *= multiplier` operator?
If there isn't could you try adding one so that this code could be the same as vector multiply all elements?